lms/djangoapps/badges/backends/tests/test_badgr_backend.py

```diff
@@ -50,21 +59,21 @@ def handler(self):
         Lazily loads a BadgeHandler object for the current course. Can't do this on setUp because the settings
         overrides aren't in place.
         """
-        return BadgeHandler(self.course.location.course_key)
+        return BadgrBackend()
 
     def test_urls(self):
         """
         Make sure the handler generates the correct URLs for different API tasks.
         """
-        self.assertEqual(self.handler.base_url, 'https://example.com/v1/issuer/issuers/test-issuer')
-        self.assertEqual(self.handler.badge_create_url, 'https://example.com/v1/issuer/issuers/test-issuer/badges')
+        self.assertEqual(self.handler._base_url, 'https://example.com/v1/issuer/issuers/test-issuer')
```

Comment:
We have several instances of this URL sprinkled around -- it should be moved into a parameter or maybe even a setting